Formula/emacs-plus@29.rb

```diff
@@ -68,12 +75,10 @@ class EmacsPlusAT29 < EmacsBase
       odie "--with-xwidgets is not available when building --with-x11"
     end
   end
-
-  #
-  # URL
-  #
-
-  url "https://github.com/emacs-mirror/emacs.git", :branch => "emacs-29"
+  
+  if build.with? "native-comp" and build.with? "native-comp-aot"
```

Comment:
ideally I would love to introduce extra option separately from 29.1 release 

p.s. what is the difference?

Reply:
I split the commit to 2 commits.  Do you still want a separate PR for the ahead-of-time commit?

Reply:
> what is the difference?

all core elisp gets compiled immediately instead of just-in-time during usage.